src/app/machines/views/MachineList/MachineListTable/NameColumn/NameColumn.tsx

```diff
@@ -51,35 +52,49 @@ const generateIPAddresses = (machine: Machine) => {
 
   if (ipAddresses.length) {
     const ipAddressesLine = (
-      <span
-        data-testid="ip-addresses"
-        title={ipAddresses.length === 1 ? ipAddresses[0] : ""}
-      >
-        {bootIP || ipAddresses[0]}
-        {ipAddresses.length > 1 ? ` (+${ipAddresses.length - 1})` : null}
-      </span>
+      <>
+        <span
+          className="u-truncate"
+          data-testid="ip-addresses"
+          title={ipAddresses.length === 1 ? ipAddresses[0] : ""}
+        >
+          {bootIP || ipAddresses[0]}
+        </span>
+      </>
     );
 
     if (ipAddresses.length === 1) {
       return ipAddressesLine;
     }
     return (
-      <Tooltip
-        message={
-          <>
-            <strong>{ipAddresses.length} interfaces:</strong>
-            <ul className="p-list u-no-margin--bottom">
-              {ipAddresses.map((address) => (
-                <li key={address}>{address}</li>
-              ))}
-            </ul>
-          </>
-        }
-        position="btm-left"
-        positionElementClassName="p-double-row__tooltip-inner"
-      >
+      <>
         {ipAddressesLine}
-      </Tooltip>
+        <Tooltip
+          message={
+            <>
+              <strong>{ipAddresses.length} interfaces:</strong>
+              <ul className="p-list u-no-margin--bottom">
+                {ipAddresses.map((address) => (
+                  <li key={address}>{address}</li>
+                ))}
+              </ul>
+            </>
+          }
+          position="right"
+          positionElementClassName="p-double-row__tooltip-inner"
+        >
+          {ipAddresses.length > 1 ? (
+            <>
+              {"("}
```

Comment:
I think this can just be `(`.

Reply:
God point, leftover from previous changes when this included a space.